### PR TITLE
Feat retrieve idfa ios14

### PR DESCRIPTION
**Pull request by IloveJavaa**
Add code for retrieve IDFA in ios 14
Asana Link - https://app.asana.com/0/0/1204397994724629/f

### CODE REVIEW

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift [hunk 1]
```diff
@@ -44,10 +45,23 @@ class TrackierSDKInstance {
         self.appToken = config.appToken
         self.installId = getInstallID()
         self.installTime = getInstallTime()
-        DispatchQueue.global().async {
-            self.trackInstall()
-            if #available(iOS 13.0, *) {
-                self.trackSession()
+        if(timeoutInterval > 0) {
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDKInstance.swift:48]: move this code to a separate function @IloveJavaa

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift [hunk 2]
```diff
@@ -44,14 +45,16 @@ class TrackierSDKInstance {
...
+            let varTimer = Timer.scheduledTimer(withTimeInterval: TimeInterval(timeoutInterval), repeats: false)
+            { (varTimer) in
+                self.idfaChecks()
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDKInstance.swift:51]:
```suggestion
                self._sendInstall()
```

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift [hunk 3]
```diff
@@ -44,14 +45,16 @@ class TrackierSDKInstance {
...
             }
+        } else {
+            idfaChecks()
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDKInstance.swift:54]:
```suggestion
            _sendInstall()
```

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift [hunk 4]
```diff
@@ -73,6 +76,15 @@ class TrackierSDKInstance {
         }
         return itd 
     }
+    
+    private func idfaChecks() {
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDKInstance.swift:80]:
```suggestion
    private func _sendInstall() {
```

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift [hunk 5]
```diff
@@ -44,14 +45,16 @@ class TrackierSDKInstance {
         self.appToken = config.appToken
         self.installId = getInstallID()
         self.installTime = getInstallTime()
-        DispatchQueue.global().async {
-            self.trackInstall()
-            if #available(iOS 13.0, *) {
-                self.trackSession()
+        if(timeoutInterval > 0) {
```
Comment (Hemant-Mann) [inline trackier-ios-sdk/Classes/TrackierSDKInstance.swift:48]:
```suggestion
        if (timeoutInterval > 0) {
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift (hunk 5 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierSDKInstance.swift:48]: done

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift (hunk 4 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierSDKInstance.swift:80]: done

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift (hunk 3 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierSDKInstance.swift:54]: done

File: trackier-ios-sdk/Classes/TrackierSDKInstance.swift (hunk 2 above)
Comment (IloveJavaa) [reply trackier-ios-sdk/Classes/TrackierSDKInstance.swift:51]: done

**Pull request merged**